docs/application/dotnet/tutorials/certificates/managing-certificate-profile.md

```diff
@@ -0,0 +1,83 @@
+
+### 1. Select an existing certificate profile
+
+Click `Select an existing certificate profile` and select a profile which you want to change the registered device then click `Next`.
+
+![Select an existing certificate profile](media/certification_guide19.png)
+
+### 2. Use the existing author certificate
+
+Click `No` to use the existing author certificate then it moves to [Create new or select existing distributor certiicate](creating-certificates.md#1-create-new-or-select-existing-distributor-certiicate) to register DUID.
+
+![Alert popup to remove author certificate](media/certification_guide20.png)
+
+> If you click `Yes`, it moves to [Create a new author certificate](creating-certificates.md#create-a-new-author-certificate) step and original author certificate will be removed permanently.
+> So it should be carefully considered.  
+> However the SDK automatically backups the original author.p12 file with timestamp extension like author.p12_bak_20160806092013 in the profile directory to protect developer from unintended removal.
+
+### 3. Create new Distributor for certificate profile
+
+Next steps are same as guided in [Create Certificate](creating-certificates.md) after [Create new or select existing distributor certiicate](creating-certificates.md#1-create-new-or-select-existing-distributor-certiicate) step
+
+
+## Importing Existing Certificates
+
+If you already have both the `author.p12` and `distributor.p12` certificates and don’t need a new certificate from Samsung, you can just import and use them to sign your application.  
+In that case, just create a new certificate profile and select existing certificates at each step.
+
+Do the same steps to create a certificate profile described in Creating Certificates before step [Create new or select old profile](creating-certificates.md#4-create-new-or-select-old-profile)
+
+### 1. Select an existing author certificate
+
+Note that the author certificate must be the same for the application to be upgraded.
+
+Select `Select an existing author certificate` and click `Next`.
+
+![Select an existing author certificate](media/certification_guide22.png)
+
+`Browse` and select your `author.p12` file and enter the password then click `Next`.
+
+![Select an existing author certificate](media/certification_guide23.png)
+
+### 2. Select an existing distributor certificate
+
+Click `Select an existing distributor certificate` and `Browse` and select your `distributor.p12` file and enter the password then click `Finish`.
+
+![Select an existing author certificate](media/certification_guide24.png)
+
+### 3. Find the created profile in the list
+
+Finally you can find the created profile in the list.
+
+![Select an existing author certificate](media/certification_guide25.png)
+
+## Troubleshooting
+
+I lost the author.p12. Can I make my application upgrade?
```

Comment:
I lost the author.p12. Can I upgrade my application?

Is it a predefined message?
If it is, please ignore my comments.

Reply:
No. I apply your sentence.